src/main/webapp/examples/team.html

```diff
@@ -0,0 +1,146 @@
+    <!-- Favicon -->
+    <link href="../assets/img/brand/favicon.png" rel="icon" type="image/png">
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
+    <!-- Icons -->
+    <link href="../assets/vendor/nucleo/css/nucleo.css" rel="stylesheet">
+    <link href="../assets/vendor/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+    <!-- Argon CSS -->
+    <link type="text/css" href="../assets/css/argon.css?v=1.0.1" rel="stylesheet">
+    <!-- Global custom CSS -->
+    <link type="text/css" href="../assets/css/global-custom.css" rel="stylesheet">
+
+</head>
+
+<body>
+<header class="header-global">
+    <!--nav bar goes here-->
+</header>
+<!-- to use the dynamic navbar, use this instead of above tag => * <div id="navbar"></div> * -->
+
+<main>
+    <section>
+        <div class="container text-center">
+            <div class="mb-5 text-left h2">
+                <span>Lorem ipsum dolor sit amet</span>
+            </div>
+
+            <div id="teamContent"></div>
+
+            <button id="btn" class="btn btn-link btn-outline-white btn-info">SEE MORE</button>
+        </div>
+    </section>
+</main>
+
+<footer class="footer">
+    <!--footer goes here -->
+</footer>
+
+<!-- to use the dynamic footer, use this instead of above tag => *<div id="footer"></div>* -->
+
+<!-- Core -->
+<script src="../assets/vendor/jquery/jquery.min.js"></script>
+<script src="../assets/vendor/popper/popper.min.js"></script>
+<script src="../assets/vendor/bootstrap/bootstrap.min.js"></script>
+<script src="../assets/vendor/headroom/headroom.min.js"></script>
+<!-- Argon JS -->
+<script src="../assets/js/argon.js?v=1.0.1"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+
+<!--Mustache template for team content-->
+<script type="text/html" id="templateTeam">
+    <div class="row">
+        {{#data}}
+        <div class="col-md-3 col-lg-3 mb-5">
+            <div class="px-4">
+                <div class="imgContainer">
+                    <img src="{{image}}"
+                         class="rounded-circle img-center img-fluid shadow shadow-lg--hover imgHover"
+                    >
+                    <div class="imgIcon">
+                        <a href="{{linkedin}}" class="btn btn-default btn-icon-only rounded-circle">
+                            <i class="fa fa-linkedin"></i>
+                        </a>
+                    </div>
+                </div>
+                <div class="pt-4 text-center">
+                    <h5 class="title">
+                        <span class="d-block mb-1">{{name}}</span>
+                        <small class="h6 text-muted">{{city}}</small>
+                    </h5>
+                </div>
+            </div>
+        </div>
+        {{/data}}
+    </div>
+</script>
+
+<!--Script for load team data-->
+<script>
+    $.ajax({
```

Comment:
wrap this with a function. call it from line 139.

Reply:
Done